packages/modals/src/styled/StyledTooltipModalBackdrop.ts

```diff
@@ -28,6 +28,11 @@ export const StyledTooltipModalBackdrop = styled.div.attrs({
   font-family: ${props => props.theme.fonts.system};
   direction: ${props => props.theme.rtl && 'rtl'};
 
+  &.garden-tooltip-modal-transition-exit-active div {
+    transition: opacity 200ms;
+    opacity: 0;
+  }
```

Comment:
This PR only involves adding the "fade-out" transition. The "fade-in" remains the same as before which leverages CSS animation from `menuStyles`.